makefile

```diff
@@ -13,7 +13,7 @@ publish:
 	pipx run nox -s publish
 
 docs:
-	pipx run nox -s docs -r
+	pipx run nox -s build_docs -r
```

Comment:
Hmm does this indicate nobody is actually using the makefile…?

Reply:
I think everything in it was migrated to nox. If it wasn’t, it should be, and the makefile should be deleted.